src/collections/_documentation/platforms/javascript/angular.md

```diff
@@ -41,14 +49,16 @@ export class AppModule {}
 
 ## AngularJS 1.x
 
-If you are using `AngularJS` `1.x` you should be able to use our AngularJS integration.
+If you're using `AngularJS 1.x`, you can use Sentry's AngularJS integration.
 
-You need to install `@sentry/integrations` with `npm` / `yarn` like:
+First, install `@sentry/integrations`:
```

Comment:
It seemed this page previously assumed the installation of `@sentry/browser` - I'm wondering if we should include that here too?